render.c

```diff
@@ -265,7 +265,6 @@ void render_frame(struct swaylock_surface *surface) {
 			// border
 			cairo_set_source_u32(cairo, state->args.colors.layout_border);
 			cairo_stroke(cairo);
-			cairo_new_sub_path(cairo);
```

Comment:
See [documentation](https://www.cairographics.org/manual/cairo-Paths.html#cairo-new-sub-path) for rationale